mysql-test/suite/rocksdb/t/use_direct_reads_writes.test

```diff
@@ -0,0 +1,47 @@
+--source include/have_rocksdb_as_default.inc
+
+# Issue221
+# Turning on both --rocksdb-allow-mmap-reads and --rocksdb-use-direct-reads
+# caused an assertion in RocksDB.  Now it should not be allowed and the
+# server will not start with that configuration
+
+# Write file to make mysql-test-run.pl expect the "crash", but don't restart
+# the server until it is told to
+--let $_server_id= `SELECT @@server_id`
+--let $_expect_file_name= $MYSQLTEST_VARDIR/tmp/mysqld.$_server_id.expect
+--exec echo "wait" >$_expect_file_name
+shutdown_server 10;
+
+# Clear the log
+--exec echo "" >$MYSQLTEST_VARDIR/log/mysqld.1.err
```

Comment:
Not sure if that does not break Valgrind memory leak checking for preceding tests in MTR. Better start with a dedicated error log here